README.md

```diff
@@ -1,29 +1,31 @@
-# Assignment 2: Weakly Supervised Object Localization
+# Assignment 1: Weakly Supervised Object Localization
 
-- [Visual Learning and Recognition (16-824) Spring 2021](https://visual-learning.cs.cmu.edu/)
-- Updated by: [Sanil Pande](https://sanilpande.github.io/)
+- [Visual Learning and Recognition (16-824) Fall 2022](https://visual-learning.cs.cmu.edu/)
+- Updated by: [Anirudh Chakravarthy](https://anirudh-chakravarthy.github.io/) and [Sai Shruthi Balaji](https://www.linkedin.com/in/sai-shruthi-balaji/)
 - Created by : [Senthil Purushwalkam](http://www.cs.cmu.edu/~spurushw/)
-- TAs: [Sanil Pande](https://sanilpande.github.io/), [Sudeep Dasari](https://sudeepdasari.github.io/), [Ziyan Wang](https://ziyanw1.github.io/)
+- TAs: [Anirudh Chakravarthy](https://anirudh-chakravarthy.github.io/), [Sai Shruthi Balaji](https://www.linkedin.com/in/sai-shruthi-balaji/), [Vanshaj Chowdhary](https://www.linkedin.com/in/vanshajchowdhary/), and [Nikos Gkanatsios](https://nickgkan.github.io/).
 
 - We will be keeping an updated FAQ on piazza. Please check the FAQ post before posting a question.
-- Due date: March 29th, 2021 at 11:59pm EST.
+- Due date: Oct 3rd, 2022 at 11:59pm EST.
 - Total points: 100
 
-In this assignment, we will learn to train object detectors in the *weakly supervised* setting, which means you're going to train object detectors without bounding box annotations!
+## Introduction
+In this assignment, we will learn to train object detectors without bounding box annotations! First, in task 1, we will use classification models and examine their backbone features for object localization cues. In task 2, we will train object detectors in the *weakly supervised* setting, which means we're going to train an object detector without bounding box annotations!
```

Comment:
"In this assignment, we will learn to train object detectors with only image level annotations, and no bounding box annotations!"

Reply:
Done